src/main/java/de/medizininformatik_initiative/process/report/ConstantsReport.java

```diff
@@ -43,7 +43,10 @@ public interface ConstantsReport
 	String BPMN_EXECUTION_VARIABLE_REPORT_RECEIVE_ERROR_MESSAGE = "reportReceiveErrorMessage";
 
 	String CODESYSTEM_REPORT = "http://medizininformatik-initiative.de/fhir/CodeSystem/report";
+
+	@Deprecated
 	String CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE = "search-bundle";
+	String CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE_V2 = "search-bundle-v2";
```

Comment:
If possible reuse existing constant and only change value.
```suggestion
	String CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE = "search-bundle-v2";
```
Maybe use the two digit process plugin version number as suffix for `search-bundle-v`. This would not introduce a new version numbering scheme and would allow the use of the placeholder `#{version}` in the Bundle StructureDefinition. The two digit process version number could be injected into a `searchBundleIdentifierValue` field of `DownloadSearchBundle` service via the `ReportConfig` using:
```
ConstantsReport.CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE + new ReportProcessPluginDefinition().getResourceVersion();
```

Reply:
Thanks for the good idea! Every time there are breaking changes in the code, we need a new bundle version. This solves the issue. Implemented via dd46ec2.